src/wad/image.rs

```diff
@@ -181,23 +215,41 @@ impl Image {
         }
     }
 
-    pub fn x_offset(&self) -> isize { self.x_offset }
-    pub fn y_offset(&self) -> isize { self.y_offset }
+    pub fn x_offset(&self) -> isize {
+        self.x_offset
+    }
```

Comment:
Could you add an empty line here, please? It works fine if they're one liners, but if they're spread out like this I think they need lines between them to breathe.